assets/base.css

```diff
@@ -1057,6 +1057,27 @@ summary::-webkit-details-marker {
   }
 }
 
+@media screen and (max-width: 989px) {
+  .grid--1-col-tablet-down .grid__item {
+    width: 100%;
+    max-width: 100%;
+  }
+
+  .slider--tablet.grid--peek.grid--1-col-tablet-down .grid__item {
+    width: 100%;
+  }
+
+  .slider--tablet.grid--peek {
+    margin: 0;
+    width: 100%;
+  }
+
+  .slider--tablet.grid--peek .grid__item {
+    box-sizing: content-box;
+    margin: 0;
+  }
```

Comment:
Here I'm just wondering what this was added for 🤔  i'm not too sure if it's needed

Reply:
This was already here (line 1152 previously). I just moved it up so it can stay in the same media query :)

Reply:
Ah ok 👍